fdbserver/Ratekeeper.actor.cpp

```diff
@@ -1202,6 +1267,17 @@ void updateRate(RatekeeperData* self, RatekeeperLimits* limits) {
 	}
 }
 
+void updateCommitCostEstimation(RatekeeperData* self, UIDTransactionTagMap<TransactionCommitCostEstimation> const& costEstimation) {
```

Comment:
Nit: Here and in a few other places, the function can be marked status. Since there are many similarly named functions across the client, proxy, and ratekeeper, IMO it is slightly easier to read through the code if the scope is explicitly limited.